### PR TITLE
J grinst patch 3

### PR DESCRIPTION
**Pull request by JGrinst**
I'm proposing here to create a new folder - within the crop models folder - for collecting all of the needed information for the Brazil Soybeans model.

There should be:
README
brazil_soybeans.ipynb
requirements.txt
__init__

### CODE REVIEW

File: api/client/samples/crop_models/brazil_soybeans/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Note: Use of Anaconda is recommended due to geopandas dependency. Please see <http://geopandas.org/install.html> for further details.
+
+### Install the API client
+
+```sh
+conda install pip
+pip install git+https://github.com/gro-intelligence/api-client.git
+```
+
+### Run prevented plant
```
Comment (itsjohnward) [inline api/client/samples/crop_models/brazil_soybeans/README.md]: Brazil soybeans*

File: api/client/samples/crop_models/brazil_soybeans/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+pandas
+matplotlib
+geopandas
+scikit-learn
+ipython
+jupyter
```
Comment (itsjohnward) [inline api/client/samples/crop_models/brazil_soybeans/requirements.txt:6]: Need to add `descartes`

**Review by itsjohnward — CHANGES_REQUESTED** [review]
Tested and working on Mac Python 2.7.16 and Windows Anaconda Python 2.7.16 with the addition of the `descartes` package mentioned below

File: api/client/samples/crop_models/brazil_soybeans/requirements.txt (hunk 2 above)
Comment (JGrinst) [reply api/client/samples/crop_models/brazil_soybeans/requirements.txt:6]: Done

File: api/client/samples/crop_models/brazil_soybeans/README.md (hunk 1 above)
Comment (JGrinst) [reply api/client/samples/crop_models/brazil_soybeans/README.md]: Done

**Review by itsjohnward — APPROVED** [review]

**Pull request merged**